assets/js/src/public.jsx

```diff
@@ -88,6 +88,7 @@ jQuery(($) => {
     const animation = form.data('animation');
     if (animation !== '') {
       formDiv.addClass(`mailpoet_form_animation_${animation}`);
+      formDiv.prev('.mailpoet_form_popup_overlay').addClass(`mailpoet_form_overlay_animation_${animation}`);
```

Comment:
I guess we could set this class in twig view https://github.com/mailpoet/mailpoet/blob/master/views/form/front_end_form.html#L10